bundles/binding/org.openhab.binding.digitalstrom/src/main/java/org/openhab/binding/digitalstrom/internal/DigitalSTROMGenericBindingProvider.java

```diff
@@ -124,6 +124,6 @@ public void removeConfigurations(String context) {
 
 	@Override
 	public Set<Item> getItemNamesByContext(String context) {
-		return new HashMap<String, Set<Item>>(super.contextMap).get(context);
+		return contextMap.get(context);
 	}
```

Comment:
Replaced creation of duplicate contextMap and extraction of set from it, with simple extraction of set from contextMap.